jax/numpy/lax_numpy.py

```diff
@@ -2250,6 +2250,20 @@ def ix_(*args):
   return tuple(output)
 
 
+@_wraps(onp.indices)
+def indices(dimensions, dtype=int32, sparse=False):
+  dimensions = tuple(dimensions)
+  N = len(dimensions)
+  output = []
+  s = dimensions
+  for i, dim in enumerate(dimensions):
+    idx = lax.iota(dtype, dim)
+    if sparse:
+      s = [1]*N
```

Comment:
Use ``ones(N, dtype=dtype)``

Reply:
on second thought, a tuple would be the right thing to use here.

Reply:
done!